web_interface/app.py

```
@@ -141,6 +141,58 @@
+            content_type = 'image/webp'
+        elif filename.lower().endswith('.svg'):
+            content_type = 'image/svg+xml'
+        elif filename.lower().endswith('.json'):
+            content_type = 'application/json'
+        elif filename.lower().endswith('.txt'):
+            content_type = 'text/plain'
+        
+        # Use send_from_directory to serve the file
+        return send_from_directory(str(assets_dir), filename, mimetype=content_type)
+        
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)